doc/source/whatsnew/v0.15.0.rst

```diff
@@ -1030,6 +1030,7 @@ Other:
   If ``Period`` freq is ``D``, ``H``, ``T``, ``S``, ``L``, ``U``, ``N``, ``Timedelta``-like can be added if the result can have same freq. Otherwise, only the same ``offsets`` can be added.
 
   .. ipython:: python
+    :okwarning:
```

Comment:
why is this needed?

Reply:
That ipython block contains the code`pd.Timdelta('120m')` that due to the current changes gives a warning because of the use of m. As a result the CI check in web and docs failed and it was suggested to add the :okwarning: line

Reply:
Today I found #6856 where it was proposed to change the ipython directives with code-blocks to prevent this issue. I have also applied that logic here